Lib/test/test_audit.py

```diff
@@ -188,5 +188,26 @@ def test_sys_getframe(self):
 
         self.assertEqual(actual, expected)
 
+    def test_syslog(self):
+        try:
+            import syslog
+        except ImportError:
+            return
```

Comment:
Use `self.skipTest()` to report the test as skipped.

Or use helper `test.support.import_helper.import_module()` to check that the module is importable.

Reply:
Good point. I should do the same with the sqlite3 test further up.